runner.yml.dist

```
@@ -14,75 +14,67 @@ drupal:
-    - "./vendor/bin/drush en config_devel -y"
-    - "./vendor/bin/drush en toolbar -y"
-    - "./vendor/bin/drush config-set system.theme default oe_theme -y"
-    - "./vendor/bin/drush config-set system.theme admin claro -y"
-    - "./vendor/bin/drush config-set node.settings use_admin_theme 1 -y"
-    - "./vendor/bin/drush config-set user.settings register visitors -y"
-    - "./vendor/bin/drush cr"
-  settings:
-    settings:
-      file_scan_ignore_directories:
-        - "node_modules"
-        - "bower_components"
-        - "vendor"
-        - "${drupal.root}"
-      file_private_path: 'sites/default/files/private'
-    databases:
-      sparql_default:
-        default:
-          prefix: ""
-          host: ${drupal.sparql.host}
-          port: ${drupal.sparql.port}
-          namespace: 'Drupal\sparql_entity_storage\Driver\Database\sparql'
-          driver: 'sparql'
+    - { task: "exec", command: "./vendor/bin/drush en oe_theme_helper -y" }
+    - { task: "exec", command: "./vendor/bin/drush theme:enable oe_theme -y" }
+    - { task: "exec", command: "./vendor/bin/drush theme:enable claro -y" }
+    - { task: "exec", command: "./vendor/bin/drush en oe_theme_content_page oe_theme_content_news oe_theme_content_policy oe_theme_content_publication oe_theme_content_event oe_theme_content_project oe_theme_content_call_tenders oe_theme_content_call_proposals oe_theme_content_consultation oe_theme_content_organisation oe_theme_content_organisation_reference oe_theme_content_person oe_theme_contact_forms oe_theme_paragraphs_contact oe_theme_paragraphs_banner oe_theme_paragraphs_carousel -y" }
+    - { task: "exec", command: "./vendor/bin/drush en oe_search oe_corporate_blocks oe_corporate_site_info field_ui config_devel toolbar -y" }
+    - { task: "exec", command: "./vendor/bin/drush en oe_multilingual -y" }
+    - { task: "exec", command: "./vendor/bin/drush config-set system.theme default oe_theme -y" }
+    - { task: "exec", command: "./vendor/bin/drush config-set system.theme admin claro -y" }
+    - { task: "exec", command: "./vendor/bin/drush config-set node.settings use_admin_theme 1 -y" }
+    - { task: "exec", command: "./vendor/bin/drush config-set user.settings register visitors -y" }
+    - { task: "exec", command: "./vendor/bin/drush cr" }
+  additional_settings: |
+        if (isset($config['system.performance']['cache']['page']['max_age'])) {
+          unset($config['system.performance']['cache']['page']['max_age']);
+        }
+        $settings['file_scan_ignore_directories'] = array('vendor', 'build', 'node_modules', 'bower_components');
+        $settings['file_private_path'] = 'sites/default/files/private';
+        $settings['extension_discovery_scan_tests'] = TRUE;
+        $databases['sparql_default']['default'] = [
+          'host' => 'sparql',
+          'port' => '8890',
+          'namespace' => 'Drupal\sparql_entity_storage\Driver\Database\sparql',
+          'driver' => 'sparql',
+        ];
```

This is what I mean.. unless there is a special code style rule for this kind of text block.
```suggestion
  additional_settings: |
    if (isset($config['system.performance']['cache']['page']['max_age'])) {
      unset($config['system.performance']['cache']['page']['max_age']);
    }
    $settings['file_scan_ignore_directories'] = array('vendor', 'build', 'node_modules', 'bower_components');
    $settings['file_private_path'] = 'sites/default/files/private';
    $settings['extension_discovery_scan_tests'] = TRUE;
    $databases['sparql_default']['default'] = [
      'host' => 'sparql',
      'port' => '8890',
      'namespace' => 'Drupal\sparql_entity_storage\Driver\Database\sparql',
      'driver' => 'sparql',
    ];
```